docs/en/grafana_template.json

```diff
@@ -15,19 +15,19 @@
   "editable": true,
   "gnetId": null,
   "graphTooltip": 0,
-  "id": 16,
-  "iteration": 1616378184198,
+  "id": null,
+  "iteration": 1640078675219,
   "links": [],
   "panels": [
     {
       "aliasColors": {},
       "bars": false,
       "dashLength": 10,
       "dashes": false,
-      "datasource": "$Source",
+      "datasource": "$datasource",
+      "description": "平均使用空间",
```

Comment:
Please use English

Reply:
I have been deleted the chinese part, add the k8s copy as well